actions/class.RestQtiTests.php

```diff
@@ -134,7 +134,7 @@ protected function getItemClassUri(): ?string
      */
     protected function isOverwriteTest(): bool
     {
-        return $this->getPostParameter(self::OVERWRITE_TEST);
+        return $this->getPostParameter(self::OVERWRITE_TEST) === 'true';
```

Comment:
What do you think about using `filter_var` here?
```php
return filter_var($this->getPostParameter(self::OVERWRITE_TEST), FILTER_VALIDATE_BOOLEAN);
```

or
```php
return filter_var($this->getPostParameter(self::OVERWRITE_TEST), FILTER_VALIDATE_BOOLEAN, FILTER_NULL_ON_FAILURE) === true;
```

Reply:
fixed